source/content/apcu.md

```diff
@@ -0,0 +1,38 @@
+---
+title: APCu Cache Basics
+description: Understand APCu and its uses within the Pantheon WebOps workflow.
+categories: [platform]
+tags: [cache, workflow]
+reviewed: "2020-10-08"
+---
+
+The **APCu** is a user-accessible PHP cache. PHP is a dynamic language that is compiled on-demand into bytecode at execution time. To improve performance, APCu stores this bytecode so that it can be reused instead of having to be recompiled each time.
+
+Pantheon provides APCu by default across all plans, but the size of the APCu memory cache ([apc.shm_size](https://www.php.net/manual/en/apcu.configuration.php#ini.apcu.shm-size")) varies depending on the service level. See the [Application Containers overview](/application-containers) to learn more about APCu on Pantheon's container architecture.
+
+## Frequently Asked Questions
+
+#### Can APCu be used as a cache backend on Pantheon?
+
+Yes, APCu can be used as a cache backend or a "key-value store"; however, this is not recommended. APCu lacks the ability to span multiple server environments. Instead, Pantheon provides [Redis](/redis) as a caching backend for Drupal and WordPress, which has coherency across multiple [application containers](/application-containers).
```

Comment:
I would say, "APCu lacks the ability to manage coherency across multiple containers." I'd like to avoid the term "environments" here because we typically use that for dev/test/live/etc.